docs/lang/articles/contribution/contributor_guide.md

```diff
@@ -43,8 +39,8 @@ Our ultimate goal is to build a sustainable, prosperous Taichi community, and ef
   - Use a spell checker, such as [Grammarly](https://app.grammarly.com/), to improve your writing in terms of grammar, style, and tone.
 
 - Constructive and courteous: Base your feedback and discussions on facts, *NOT* on personal feelings.
-  - Acceptable: *"This design could be confusing to new Taichi users. If it were designed this way, it could..."*
-  - Undesirable: *"This design is terrible."*
+  - Acceptable😃: *"This design could be confusing to new Taichi users. If it were designed this way, it could..."*
+  - Undesirable😞: ~~*"This design is terrible."*~~
 
 ## What can you contribute
```

Comment:
```suggestion
## What you can contribute
```
(Not entirely sure about this, so feel free to reject this proposal.)

Reply:
I had the same question when I was writing this title. Let's use assertive sentence in a question title.